### DISCUSSION TITLE
customizing wallpaper at the end of selenium video

### DISCUSSION THREAD

**Discussion by johndaniel7** [SeleniumHQ/docker-selenium#2378] [Q&A] [answered]
can i customize the image at the end of the selenium video recording, as of now it is a selenium grid logo. but can i customize it this [docker compose file](https://github.com/SeleniumHQ/docker-selenium/blob/trunk/docker-compose-v3-video-upload.yml)?                                    Thanks

**Comment by VietND96** [comment, marked as answer]
I think via volume, you can mount another image to /usr/share/images/fluxbox/ubuntu-light.png in container
